dll/win32/newdev/wizard.c

```diff
@@ -579,6 +579,32 @@ AutoDriver(HWND Dlg, BOOL Enabled)
     IncludePath(Dlg, Enabled & IsDlgButtonChecked(Dlg, IDC_CHECK_PATH));
 }
 
+static INT CALLBACK
+BrowseCallbackProc(HWND hwnd, UINT uMsg, LPARAM lParam, LPARAM lpData)
```

Comment:
I'm not sure it's correct to use `static` attribute for a callback procedure.

Reply:
Here, static means that the function "visibility" (by name) will be limited to the .c file where it is. But it doesn't change the fact that it's a "callback" (stdcall) function that can be passed around.